### PR TITLE
[WIP] Calendar

### PR DESCRIPTION
**Pull request by mattwalo32**

### CODE REVIEW

**Comment by SirajChokshi** [comment]
Screen Shot 2020-02-16 at 2 41 53 PM

**Comment by mattwalo32** [comment]
Calendar Screen

File: App/Components/AddBillDialog.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
+// /*This is an Example of PopuUp Dialog Example in React Native*/
```
Comment (uawasth2) [inline App/Components/AddBillDialog.js]: Delete, if not being used.

**Comment by mattwalo32** [comment]
Calendar

**Review by uawasth2 — CHANGES_REQUESTED** [review]

**Review by uawasth2 — APPROVED** [review]

**Pull request merged**